.github/workflows/Benchmarks.yml

```diff
@@ -1,13 +1,21 @@
 name: Benchmarks
 on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
```

Comment:
Do we really want to run the Benchmarks only after CI is finished? This would mean more time to wait for the actions before a PR can be merged. Currently, the benchmarks run for ~3 minutes, but this might increase in the future and then could become annoying to wait.

Reply:
My thought is that this saves a potentially pointless Benchmarks run if failing tests would cause failures in the benchmark.

This becomes more important in a future with a more substantial benchmark suite: Generally the only benchmarks that matter for a given PR are benchmarks that test new or newly modified code; if the test suite fails, then the (specifically relevant) benchmarks won't finish/succeed either.

IMO that's a reasonable tradeoff for a slightly delayed merge timeframe.